gpbft/powertable.go

```diff
@@ -79,6 +79,17 @@ func (p *PowerTable) Get(id ActorID) (*StoragePower, PubKey) {
 	return nil, nil
 }
 
+// GetRelative returns the relative power of a specific participant.
+func (p *PowerTable) GetRelative(id ActorID) *big.Float {
```

Comment:
Ditto comment about no floats. Just return the numerator and denominator both so the caller can use integer arithmetic.

Reply:
This can be removed now.

Reply:
Done.